### PR TITLE
ruby: add Indonesian translation

### PR DESCRIPTION
**Pull request by apraditya**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.id/common/ruby.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Interpreter bahasa pemrograman Ruby.
+> Informasi lebih lanjut: <https://www.ruby-lang.org>.
+
+- Memulai REPL (_shell_ interaktif):
+
+`irb`
+
+- Menjalankan skrip Ruby:
+
+`ruby {{script.rb}}`
```
Comment (blueskyson) [inline pages.id/common/ruby.md:12]:
```suggestion
`ruby {{path/to/script.rb}}`
```

File: pages.id/common/ruby.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Menjalankan sebuah perintah Ruby dalam _command-line_:
+
+`ruby -e {{command}}`
+
+- Memeriksa kesalahan sintaks dari skrip Ruby:
+
+`ruby -c {{script.rb}}`
```
Comment (blueskyson) [inline pages.id/common/ruby.md:20]:
```suggestion
`ruby -c {{path/to/script.rb}}`
```

File: pages.id/common/ruby.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# ruby
+
+> Interpreter bahasa pemrograman Ruby.
+> Informasi lebih lanjut: <https://www.ruby-lang.org>.
+
+- Memulai REPL (_shell_ interaktif):
```
Comment (blueskyson) [inline pages.id/common/ruby.md:6]: I'm not sure if we allow `_something_` syntax in tldr.

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution.

File: pages.id/common/ruby.md (hunk 3 above)
Comment (navarroaxel) [reply pages.id/common/ruby.md:6]: Yeah, we accept that. We don't have an strict rule and I understand the author uses this for English words.

Comment (apraditya) [reply pages.id/common/ruby.md:6]: If doesn't throw any error, do you think it's still okay to leave it like that until `tldr` supports the syntax? I mean it's still a valid markdown. wdyt?

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**